homeassistant/components/reolink/__init__.py

```diff
@@ -79,6 +79,19 @@ async def async_device_config_update():
                     f"Error updating Reolink {host.api.nvr_name}"
                 ) from err
 
+        async with async_timeout.timeout(host.api.timeout):
+            try:
+                await host.renew()
+                host.lost_subscription = False
```

Comment:
Please only wrap the line that can raise inside the try... except block.

Reply:
done